recipes/plothist/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "plothist" %}
+{% set version = "1.2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/plothist-{{ version }}.tar.gz
+  sha256: b4606d9ff7db3cc1a7f6f5d5440fd959e1d1292818910cc0d142b23e65e4de76
+
+build:
+  entry_points:
+    - install_latin_modern_fonts = plothist.scripts.install_latin_modern_fonts:install_latin_modern_fonts
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - flit-core >=3.2,<4
+    - pip
+  run:
+    - python >=3.7
+    - boost-histogram >=1.4.0,<1.5.dev0
+    - numpy >=1.14.5
+    - matplotlib-base >=3.0
+    - pyyaml >=5.3.1
+    - scipy >=1.6.0
+    # Needed for install_latin_modern_fonts
+    - wget
+    - unzip
```

Comment:
c.f. https://github.com/cyrraz/plothist/issues/175

Reply:
Should we wait for https://github.com/cyrraz/plothist/pull/176 to be merged and a new small release? Or is it ok to be merged now?

Reply:
Conda-forge will create new release PRs automatically on the `conda-forge/plothist-feedstock` that will be created once this is merged, so no real need here to wait now. You can just remove those dependencies in the following release PR.

Reply:
I am not familiar with the workflow of conda-forge. We created the new release v1.2.6, but not PR got automatically created in https://github.com/conda-forge/plothist-feedstock. Should I do what the readme says (fork + PR) or did we miss a setup that would automatically create the PR?

Reply:
Thank you, it worked perfectly https://github.com/conda-forge/plothist-feedstock/pull/1 :smile: